suite-native/module-settings/src/components/ViewOnly/WalletRememberModeIconButton.tsx

```diff
@@ -0,0 +1,162 @@
+        }
+    };
+
+    const handleDisableViewOnly = () => {
+        showAlert({
+            title: (
+                <Translation
+                    id="moduleSettings.viewOnly.disableDialog.title"
+                    values={{ name: walletNameLabel }}
+                />
+            ),
+            description: (
+                <Translation
+                    id="moduleSettings.viewOnly.disableDialog.subtitle"
+                    values={{ device: deviceLabel }}
+                />
+            ),
+            primaryButtonTitle: (
+                <Translation id="moduleSettings.viewOnly.disableDialog.buttons.primary" />
+            ),
+            onPressPrimaryButton: toggleViewOnly,
+            primaryButtonVariant: 'redBold',
+            secondaryButtonTitle: (
+                <Translation id="moduleSettings.viewOnly.disableDialog.buttons.secondary" />
+            ),
+            onPressSecondaryButton: hideAlert,
+            secondaryButtonVariant: 'redElevation0',
+        });
+    };
+
+    const handleEjectWallet = () => {
+        const settings: ConnectDeviceSettings = {
+            defaultWalletLoading: 'standard',
+        };
+
+        analytics.report({
+            type: EventType.ViewOnlyChange,
+            payload: { enabled: !device.remember, origin: 'settingsToggle' },
+        });
+
+        if (device.connected) {
+            dispatch(toggleRememberDevice({ device }));
+            if (device.remember) {
+                showToast({
+                    variant: 'default',
+                    message: (
+                        <Translation id="moduleSettings.viewOnly.autoEject.toast.walletsWillBeEjected" />
+                    ),
+                });
+            }
+        } else {
+            dispatch(deviceActions.forgetDevice({ device, settings }));
+            showToast({
+                variant: 'default',
+                message: <Translation id="moduleSettings.viewOnly.autoEject.toast.walletEjected" />,
+            });
+        }
+    };
+
+    if (!isViewOnlyByDefaultEnabled) {
+        return (
+            <Button
+                size="extraSmall"
+                colorScheme={device.remember ? 'redElevation0' : 'primary'}
+                onPress={() => (device.remember ? handleDisableViewOnly() : toggleViewOnly())}
+                testID={`@settings/view-only/toggle-button/${device.features?.label}/${device.walletNumber ?? 0}`}
+            >
+                <Translation
+                    id={
+                        device.remember
+                            ? 'moduleSettings.viewOnly.button.disable'
+                            : 'moduleSettings.viewOnly.button.enable'
+                    }
+                />
+            </Button>
+        );
+    }
+
+    if (isDeviceAutoEjectEnabled) return null;
+
+    return (
+        <Animated.View entering={FadeIn} exiting={FadeOut}>
+            <IconButton
```

Comment:
`size="extraSmall"` is missing, I guess?

Reply:
https://github.com/trezor/trezor-suite/pull/20300/commits/da7d6f6b00dd3e4f278b82deda58b04f6cae4199